lib/phoenix_live_view/test/client_proxy.ex

```diff
@@ -456,6 +456,9 @@ defmodule Phoenix.LiveViewTest.ClientProxy do
 
       :error ->
         case Map.fetch(state.dropped_replies, ref) do
+          {:ok, :missing_from} ->
+            {:noreply, %{state | dropped_replies: Map.delete(state.dropped_replies, ref)}}
+
           {:ok, from} ->
             GenServer.reply(from, {:ok, nil})
```

Comment:
Maybe this:

```suggestion
          {:ok, from} ->
            from && GenServer.reply(from, {:ok, nil})
```

and then revert to the previous code below?

Reply:
Updated the code.

Just thought that adding `:missing_from` will explicitly show that we have no "from".

Agree, that this approach `from && GenServer.reply(from, {:ok, nil})` is simpler, and easier to support.